### PR TITLE
Reboot with bootstrap user in ci-aio env

### PR DESCRIPTION
**Pull request by Alex-Welsh**

### CODE REVIEW

File: etc/kayobe/environments/ci-aio/reboot.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+# Ensure that the reboot playbook is always executed using the boostrap user
+reboot_with_bootstrap_user: true
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-aio/reboot.yml]: Can it not be applied at the import_playbook level?

Comment (Alex-Welsh) [reply etc/kayobe/environments/ci-aio/reboot.yml]: I like the idea, I'll try it out

**Review by markgoddard — APPROVED** [review]
LGTM if it works

**Comment by Alex-Welsh** [comment]
https://github.com/stackhpc/stackhpc-kayobe-config/pull/1281
works :)

**Pull request merged**